src/libopenrave/planner.cpp

```diff
@@ -99,7 +99,16 @@ int AddStatesWithLimitCheck(std::vector<dReal>& q, const std::vector<dReal>& qde
     return status;
 }
 
-PlannerStatus::PlannerStatus() : statusCode(0)
+PlannerStatus::PlannerStatus() {}
+
+// To maintain backwards compatibility...
+PlannerStatus::PlannerStatus(const std::string& description, const int statusCode, CollisionReportPtr report) : description(description), statusCode(statusCode)
+{
+    // not sure if this works...
```

Comment:
No, it does not. `__FILE__` is a macro resolved at compile time, it will always be `planner.cpp`

Reply:
Oh, I actually just copied the function to maintain backwards compability. I'll delete it.